tools/tidy

```diff
@@ -32,62 +39,56 @@ while true; do
   esac
 done
 
-cleanup() {
-    find . -name '*.tdy' -delete
-}
-
 trap cleanup EXIT
 
 if ! command -v perltidy > /dev/null 2>&1; then
     echo "No perltidy found, install it first!"
     exit 1
 fi
 
+# cpan file is in top directory
+dir="$(dirname $(readlink -f $0))/.."
 perltidy_version_found=$(perltidy -version | sed -n '1s/^.*perltidy, v\([0-9]*\)\s*$/\1/p')
-perltidy_version_expected=$(sed -n "s/^.*Tidy[^0-9]*\([0-9]*\)['];$/\1/p" "${0%/*}/../cpanfile")
+perltidy_version_expected=$(sed -n "s/^.*Tidy[^0-9]*\([0-9]*\)['];$/\1/p" $dir/cpanfile)
 if [ "$perltidy_version_found" != "$perltidy_version_expected" ]; then
     echo "Wrong version of perltidy. Found '$perltidy_version_found', expected '$perltidy_version_expected'"
     exit 1
 fi
 
-find_changed_files() {
-    changed_files=()
-    for file in "${files[@]}"; do
-        if ! [ $only_changed ] || [ "$(git diff --name-only "$file")" ]; then
-            changed_files+=("$file")
-        fi
-    done
+find-files() {
+    local files=('**.p[ml]' '**.t')
+    if [[ -d script ]]; then
+        files+=($(file --mime-type script/*| grep perl | awk -F':' '{ print $1 }'))
+    fi
+    if [[ -f isotovideo ]]; then
```

Comment:
Maybe we can go by `file --mime-type` here, too?

Reply:
What do you mean? Something like
`file --mime-type *`
?

Reply:
We can assume if it's `text-x-perl` we want to check it, like the `script` folder just above, no?

Reply:
Ok, I rewrote that part.